src/libpcp_web/src/query.c

```diff
@@ -190,6 +190,8 @@ freeSeriesGetSID(seriesGetSID *sid)
 
     seriesBatonCheckMagic(sid, MAGIC_SID, "freeSeriesGetSID");
     sdsfree(sid->name);
+    if (sid->metric)
```

Comment:
No need to check for null here - sdsfree() does this.  I'll tweak it when merging.